nrf_security/Kconfig.tls

```diff
@@ -439,3 +439,9 @@ endmenu
 
 
 endif # MBEDTLS_TLS_LIBRARY
+
+config OPENTHREAD_MBEDTLS_DEBUG
+	bool "MbedTLS logs for OpenThread"
+	select MBEDTLS
```

Comment:
Maybe this should rather be depends on MBEDTLS?

Reply:
This makes sense, but this config is disabled in our configuration (and is promptless). I believe we need to select it.